type_test.go

```diff
@@ -0,0 +1,105 @@
+package hubspot_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/belong-inc/go-hubspot"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestHsStr_String(t *testing.T) {
+	tests := []struct {
+		name string
+		hs   *hubspot.HsStr
+		want string
+	}{
+		{
+			name: "Success",
+			hs:   hubspot.NewString("text"),
+			want: "text",
+		},
+		{
+			name: "Success case of nil receiver",
```

Comment:
In Addition, is  `Success case` required? Where is `Failed case`? Name as what it is doing like `ReceiveIsNil` would be enough.